.TOOLVERSIONS

```diff
@@ -1,6 +1,6 @@
-android-ndk;r10e;070be287539e3e7706f8dabfb6bf9879
-android-sdk-build-tools;28.0.1;
-android-sdk-platform;android-27;
+android-ndk;r17c;a4b6b8281e7d101efd994d31e64af746
+android-sdk-build-tools;28.0.3;
+android-sdk-platform;android-28;
 android-sdk;4333796;aa190cfd7299cd6a1c687355bb2764e4
```

Comment:
should this line be changed? how do we manage this file btw? is this some docker related stuff?

Reply:
This file is read when generating the Dockerfile to figure out which files to incorporate in it. It is also used to tag the Docker image with a different hash, depending on the values we put here.

Reply:
ok, understood